rmk/src/combo.rs

```diff
@@ -59,17 +61,39 @@ impl Combo {
             }
         }
 
+        debug!("combo {:?} search key action {:?} ", self, key_action);
         let action_idx = self.actions.iter().position(|&a| a == key_action);
         if let Some(i) = action_idx {
             self.state |= 1 << i;
-        } else if !self.done() {
+            debug!("combo {:?} found index {} updated state: {}", self, i, self.state);
+        } else if !self.satisfy() {
             self.reset();
+            debug!("combo {:?} reset state: {}", self, self.state);
         }
         action_idx.is_some()
     }
 
+    pub(crate) fn mark_done(&mut self) -> KeyAction {
```

Comment:
Adding some documentation about `mark_done`, `done` and `satisfy` would be great